src/app/shared/form/form.service.spec.ts

```diff
@@ -40,34 +35,36 @@ describe('FormService test suite', () => {
         required: 'Title is required'
       }
     }),
-    new DynamicInputModel({id: 'date'}),
-    new DynamicInputModel({id: 'description'}),
-    new DynamicFormGroupModel({
-
-      id: 'addressLocation',
-      group: [
-        new DynamicInputModel({
-
-          id: 'zipCode',
-          label: 'Zip Code',
-          placeholder: 'ZIP'
-        }),
-        new DynamicInputModel({
-
-          id: 'state',
-          label: 'State',
-          placeholder: 'State'
-        }),
-        new DynamicInputModel({
-
-          id: 'city',
-          label: 'City',
-          placeholder: 'City'
-        })
-      ]
-    }),
+    new DynamicInputModel({ id: 'date' }),
+    new DynamicInputModel({ id: 'description' }),
+    // new DynamicFormGroupModel({
```

Comment:
Unclear why this entire section of the spec has been commented out.  Could we add a comment here? Or should this be removed for some reason?